SR2021/2020-11-13-sr2021-kickstart-details.md

```diff
@@ -0,0 +1,38 @@
+---
+to: Potential team leaders + signed up teams
+subject: Student Robotics Kickstart Information
+---
+
+Hello there!
+
+We're pleased to announce some more details for the Student Robotics 2020 kickstart.
+
+As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.
+
+## Kickstart
+
+Kickstart will be streamed live on **21st November 2020 at 13:00 GMT**, aiming to finish around 16:00. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
+
+The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/simulator/) during the event.
```

Comment:
Would be good to add something here acknowledging that this is the same simulator as the virtual competition ran in, but that users should download the latest version.

Reply:
Perhaps
```suggestion
The simulator will be based in [Webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. If you attended last year's virtual competition and still have Webots installed, you should upgrade to the latest version as there have been a number of bug fixes.

Our simulation (containing the robot and the arena) is a separate, much smaller, download which will be made available on [the docs](https://studentrobotics.org/docs/simulator/) during the event.
```

Reply:
Oh, just spotted you'd also some wording.

Reply:
https://github.com/srobo/team-emails/pull/66/commits/b4284812eeb5c6ebbe78608cead8b1eb906d84f4